src/test/java/com/epam/page/object/generator/integrationalTests/CustomClassLoader.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.page.object.generator.integrationalTests;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public class CustomClassLoader extends ClassLoader {
+    private static final String CLASS_EXTENSION = ".class";
+    private static final String TEST_PACKAGE = "test";
+    private static final String MANUAL_PACKAGE = "manual";
+
+    @Override
+    public Class<?> loadClass(String name) throws ClassNotFoundException {
+        if (name.startsWith(TEST_PACKAGE) || name.startsWith(MANUAL_PACKAGE)) {
+            try {
+                InputStream is = getParent().getResourceAsStream(
```

Comment:
1. Please, use IOUtils.toByteArray(InputStream) here. 
2. Please also note that any input stream must be closed in finally block or by using try-with-resources.
3. Instead of empty string in ClassNotFoundException, use name of class that was not found.
4. I wonder why you use custom class loader, can you provide explanation for this decision? Add your answer to javadoc of the class.

Reply:
fixed. (additional: should we add java doc comments to all method/classes? Or only in places that you've mentioned?)